polybar-scripts/anyconnect-status/vpnstatus.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Could you use `/bin/sh` and fix all `shellsheck` errors? I don't see why bash is explicitly needed here.